src/components/calcite-inline-editable/calcite-inline-editable.e2e.ts

```diff
@@ -253,7 +253,7 @@ describe("calcite-inline-editable", () => {
 
     it("disables editing when afterConfirm resolves successfully", async () => {
       const element = await page.find("calcite-inline-editable");
-      const afterConfirm = () => new Promise((resolve) => setTimeout(resolve, 100));
+      const afterConfirm: () => Promise<void> = () => new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
Nice 👍

Reply:
yeah this kept popping up randomly. needed a type